lib/src/view/user/perf_stats_screen.dart

```diff
@@ -233,15 +234,15 @@ class _Body extends ConsumerWidget {
                         child: _RatingWidget(
                           data.highestRating,
                           data.highestRatingGame,
-                          context.lichessColors.good,
+                          LichessColors.good,
```

Comment:
This was done so for a reason, please don't remove things before understanding why.
Explanation: https://github.com/lichess-org/mobile/blob/main/lib/src/theme.dart#L155

this applies to all the changes here.